stdlib/_operator.pyi

```diff
@@ -1,64 +1,129 @@
 import sys
-from operator import (
-    abs as abs,
-    add as add,
-    and_ as and_,
-    attrgetter as attrgetter,
-    concat as concat,
-    contains as contains,
-    countOf as countOf,
-    delitem as delitem,
-    eq as eq,
-    floordiv as floordiv,
-    ge as ge,
-    getitem as getitem,
-    gt as gt,
-    iadd as iadd,
-    iand as iand,
-    iconcat as iconcat,
-    ifloordiv as ifloordiv,
-    ilshift as ilshift,
-    imatmul as imatmul,
-    imod as imod,
-    imul as imul,
-    index as index,
-    indexOf as indexOf,
-    inv as inv,
-    invert as invert,
-    ior as ior,
-    ipow as ipow,
-    irshift as irshift,
-    is_ as is_,
-    is_not as is_not,
-    isub as isub,
-    itemgetter as itemgetter,
-    itruediv as itruediv,
-    ixor as ixor,
-    le as le,
-    length_hint as length_hint,
-    lshift as lshift,
-    lt as lt,
-    matmul as matmul,
-    methodcaller as methodcaller,
-    mod as mod,
-    mul as mul,
-    ne as ne,
-    neg as neg,
-    not_ as not_,
-    or_ as or_,
-    pos as pos,
-    pow as pow,
-    rshift as rshift,
-    setitem as setitem,
-    sub as sub,
-    truediv as truediv,
-    truth as truth,
-    xor as xor,
+from typing import (
+    Any,
+    AnyStr,
+    Callable,
+    Container,
+    Generic,
+    Mapping,
+    MutableMapping,
+    MutableSequence,
+    Sequence,
+    SupportsAbs,
+    Tuple,
+    TypeVar,
+    overload,
 )
-from typing import AnyStr
+from typing_extensions import ParamSpec, SupportsIndex, final
+
+_R = TypeVar("_R")
+_T = TypeVar("_T")
+_T_co = TypeVar("_T_co", covariant=True)
+_K = TypeVar("_K")
+_V = TypeVar("_V")
+_P = ParamSpec("_P")
+
+def lt(__a: Any, __b: Any) -> Any: ...
+def le(__a: Any, __b: Any) -> Any: ...
+def eq(__a: Any, __b: Any) -> Any: ...
+def ne(__a: Any, __b: Any) -> Any: ...
+def ge(__a: Any, __b: Any) -> Any: ...
+def gt(__a: Any, __b: Any) -> Any: ...
+def not_(__a: Any) -> bool: ...
+def truth(__a: Any) -> bool: ...
+def is_(__a: Any, __b: Any) -> bool: ...
+def is_not(__a: Any, __b: Any) -> bool: ...
+def abs(__a: SupportsAbs[_T]) -> _T: ...
+def add(__a: Any, __b: Any) -> Any: ...
+def and_(__a: Any, __b: Any) -> Any: ...
+def floordiv(__a: Any, __b: Any) -> Any: ...
+def index(__a: Any) -> int: ...
+def inv(__a: Any) -> Any: ...
+def invert(__a: Any) -> Any: ...
+def lshift(__a: Any, __b: Any) -> Any: ...
+def mod(__a: Any, __b: Any) -> Any: ...
+def mul(__a: Any, __b: Any) -> Any: ...
+def matmul(__a: Any, __b: Any) -> Any: ...
+def neg(__a: Any) -> Any: ...
+def or_(__a: Any, __b: Any) -> Any: ...
+def pos(__a: Any) -> Any: ...
+def pow(__a: Any, __b: Any) -> Any: ...
+def rshift(__a: Any, __b: Any) -> Any: ...
+def sub(__a: Any, __b: Any) -> Any: ...
+def truediv(__a: Any, __b: Any) -> Any: ...
+def xor(__a: Any, __b: Any) -> Any: ...
+def concat(__a: Sequence[_T], __b: Sequence[_T]) -> Sequence[_T]: ...
+def contains(__a: Container[Any], __b: Any) -> bool: ...
+def countOf(__a: Container[Any], __b: Any) -> int: ...
```

Comment:
This seems wrong since Container only provides `__contains__`. I'm guessing this needs `__getitem__` or even `.count()`. We can fix that in a separate PR though.